docbook/gnc-docbookx.dtd

```diff
@@ -165,6 +165,6 @@ own entity definitions to use across all of our DocBook pages. -->
 
 <!-- c) URLs of Authors -->
 <!ENTITY url-selinger "https://www.mathstat.dal.ca/~selinger/">
-<!ENTITY tit-sel-tut "&url-selinger;accounting/tutorial.html">
-<!ENTITY tit-sel-gnc "&url-selinger;accounting/gnucash.html">
+<!ENTITY tit-sel-tut "<foreignphrase>Tutorial on multiple currency accounting</foreignphrase>">
+<!ENTITY tit-sel-gnc "<foreignphrase>Multiple currency accounting in GnuCash</foreignphrase>">
```

Comment:
Wenn du `<foreignphrase>` hier einfügst, solltest du sie im pt Text entfernen.

Ich würde ja mal
`grep -inr '&url-selinger;' manual guide`
ausführen und alle Vorkommen überprüfen. Es wäre bei der Gelegenheit auch sinnvoll, `accounting/`in die Entität verschieben,so daß die Vorkommen dann
`<ulink url="&url-selinger;tutorial">&tit-sel-tut;</ulink>` und  
`<ulink url="&url-selinger;gnucash">&tit-sel-gnc;</ulink>` lauten sollten.